idunn/places/poi.py

```diff
@@ -12,6 +13,19 @@ def __init__(self, d):
             self["properties"] = {p.get("key"): p.get("value") for p in self.get("properties", [])}
         self.properties = self["properties"]
 
+        # If it's a trash can and in Bretagne (in France), we get its volume and the time of the
+        # last update
+        if (self.properties.get("amenity", "") == "recycling"
+            and any(x.get("label", "").lower().endswith("bretagne, france")
+                    for x in d.get("administrative_regions", []))):
```

Comment:
It should be possible to use the different `codes` present in each administrative region, instead of relying on string comparison.  

For example for Finistère:
```
"codes": [
  {"name":"ISO3166-2","value":"FR-29"},
  {"name":"ref:INSEE","value":"29"},
  {"name":"ref:NUTS","value":"FR522"},
  {"name":"wikidata","value":"Q3389"}
]
```
The `wikidata` entity or the `ISO3166-2` code could be used as reference.

Reply:
I found this way of doing it simpler and easier to read.